templates/zest/templates.yaml

```diff
@@ -0,0 +1,140 @@
+dataset: zest
+templates:
+  223c6226-5f2d-4dd8-9710-4657ffb54f13: !Template
+    id: 223c6226-5f2d-4dd8-9710-4657ffb54f13
+    jinja: '{{context}}
+
+      {{question}}|||
+
+      {% if answer[0] == "n/a" %}
+
+      I don''t know
+
+      {% else %}
+
+      answer[0]
+
+      {% endif %}'
+    name: concatenate
+    reference: Concatenate question and question
+  2283cebf-988e-4bff-96bf-982a09963e49: !Template
+    id: 2283cebf-988e-4bff-96bf-982a09963e49
+    jinja: 'Decide whether the question "{{question}}" is answerable solely based
+      on this passage: {{context}}|||
+
+      {% if answer[0] == "n/a" %}
+
+      No
+
+      {% else %}
+
+      Yes
+
+      {% endif %}'
+    name: answerable_or_not
+    reference: Decide whether this question is answerable
+  6f694e45-1d17-4067-a1f6-7dae89c148db: !Template
+    id: 6f694e45-1d17-4067-a1f6-7dae89c148db
+    jinja: 'My daughter is asking me a question about {{domain}}: {{question}}
+
+      Here''s what I found on the internet: {{context}}
+
+      What''s the answer?
+
+      Answer: |||
+
+      {% if answer[0] == "n/a" %}
+
+      Can''t answer
+
+      {% else %}
+
+      answer[0]
+
+      {% endif %}'
+    name: curious_kid
+    reference: Answer the questions of a curious kid
+  7425232a-9880-428c-9ddc-4070e50e22cc: !Template
+    id: 7425232a-9880-428c-9ddc-4070e50e22cc
+    jinja: 'Answer the question based on the context. If the question is not answerable
+      with the context alone, say "can''t answer".
+
+      {{context}}
+
+      {{question}}|||
+
+      {% if answer[0] == "n/a" %}
+
+      Can''t answer
+
+      {% else %}
+
+      answer[0]
+
+      {% endif %}'
+    name: gpt3instruct_format
+    reference: Template format from GPT3 instruct
+  846cc8ff-0527-4b2f-8da4-46613e915ff5: !Template
+    id: 846cc8ff-0527-4b2f-8da4-46613e915ff5
+    jinja: '{% if answer[0] != "n/a" %}{{context}}
+
+      Based on the previous passage, generate a question which has for answer {{answer[0]}}.
+
+      |||
+
+      {{question}}
+
+      {% endif %}'
+    name: generate_the_question
+    reference: Generate the question
+  a69d7845-8503-48c4-b3d2-17bdc6820794: !Template
+    id: a69d7845-8503-48c4-b3d2-17bdc6820794
+    jinja: '{% if answer[0] != "n/a" %}Generate a question about {{domain}} based
+      on this passage: {{context}}.
+
+      The answer to the question should be: {{answer[0]}}.
+
+      |||
+
+      {{question}}
+
+      {% endif %}'
+    name: generate_the_question_with_domain
+    reference: Generate the question with domain
+  bdaf4f8a-2344-4e46-a52b-2045a080a4b2: !Template
+    id: bdaf4f8a-2344-4e46-a52b-2045a080a4b2
+    jinja: 'Answer this question about {{domain}} based on the context. If the question
```

Comment:
domain is of the form "national_parks" I think you could replace the "_" with a space?